### DISCUSSION TITLE
Getting Started: Start-Up Command Fails

### DISCUSSION THREAD

**Discussion by coreybyrum7** [epicweb-dev/epic-stack#585] [Q&A] [answered]
Attempting to follow [getting-started.md](https://github.com/epicweb-dev/epic-stack/blob/main/docs/getting-started.md)

## Steps
- Ran `npx create-epic-app@latest`

❌ Error received
```javascript
user@machine epic-stack % npx create-epic-app@latest

 remix   v2.4.1 💿 Let's build a better website...

   dir   Where should we create your new project?
         /test-website

      ◼  Template: Using epicweb-dev/epic-stack...
      ✔  Template copied
node:internal/errors:866
  const err = new Error(message);
              ^

Error: Command failed: npx --yes create-remix@latest --install --no-git-init --init-script --template epicweb-dev/epic-stack
    at checkExecSyncError (node:child_process:890:11)
    at execSync (node:child_process:962:15)
    at createEpicApp (file:///Users/user/.npm/_npx/cac6a35e04d35842/node_modules/create-epic-app/index.js:7:3)
    at file:///Users/coreybyrum/.npm/_npx/cac6a35e04d35842/node_modules/create-epic-app/index.js:10:1
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 38446,
  stdout: null,
  stderr: null
}

Node.js v20.9.0
```

Any help would be appreciated!

**Comment by kentcdodds** [comment]
I'm a little confused by this step:

> Pulled the repo onto my local file system

You shouldn't have to pull any repo. Simply run `npx create-epic-app@latest` wherever you want the project to go. It will ask you for a name and will create the directory for you.

**Comment by coreybyrum7** [comment]
I see! Apologies! I'm fairly new to Remix and utilizing templates/stacks.

Unfortunately, the issue above still exists for me when running `npx create-epic-app` directly from an arbitrary folder (not a repo). I can update the description to reflect the issue.

Could it be my node version? What's the expected node range for `create-epic-app`?

**Comment by kentcdodds** [comment]
That's interesting. It seems to be working fine for me. I'm on node v20.10.0 so I don't expect that to be the issue. It does appear that your error is coming from something internal to node. Unfortunately it doesn't really look like there's much helpful info in the error message. Sorry! Maybe someone else has seen this before 🙏

**Comment by coreybyrum7** [comment, marked as answer]
Coming back to this with an explanation.. 

`create-remix` cli wasn't installed. Perhaps an implicit assumption in the getting-started docs. I had tried installing originally from a new machine without the remix cli installed.

**Reply by kiliman** [reply]
Hmm... `create-epic-app` calls `npx --yes create-remix@latest`.

`npx` will automatically install the `create-remix` package if it doesn't exist. Unfortunately, the error you're getting doesn't give much detail.

Did you try the `create-epic-app` command multiple times? I wonder if it was just a temporary error, perhaps with the network.

**Reply by coreybyrum7** [reply]
Thanks for your reply! It ended up working after running the remix-indie template elsewhere, which asked to install the CLI before continuing.
